packages/tasks/src/model-libraries.ts

```diff
@@ -372,6 +372,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		countDownloads: {
 			wildcard: { path: "*.sentis" },
 		},
+	"whisperkit": {
+		prettyLabel: "WhisperKit",
+		repoName: "WhisperKit",
+		repoUrl: "https://github.com/argmaxinc/WhisperKit",
+		countDownloads: {
+			wildcard: { path: "**/model.mil" },
```

Comment:
```suggestion
			wildcard: { path: "*/model.mil" },
```

I expect this to be the correct way for ES wildcards

Reply:
```shell
ati@atis-MBP-2 whisperkit-coreml % ls -al **/model.mil | wc -l
      42
ati@atis-MBP-2 whisperkit-coreml % ls -al */model.mil | wc -l 
zsh: no matches found: */model.mil
       0
```

Reply:
The wildcard pattern passed here correspond to an Elasticsearch string filter. This means the whole filepath is considered as a string, which is not the same as in `ls` command.